File: Desafios em C/desafio1048.c
#include <stdio.h>

int main (){


    float salario, aumento, novoSalario;

    scanf("%f", &salario);

    if(salario >= 0 && salario <=400){
        novoSalario = salario + (salario * (15.0/100.0));
        aumento = salario * (15.0/100.0);       
        printf("Novo salario: %.2f\n",novoSalario);
        printf("Reajuste ganho: %.2f\n", aumento);
        printf("Em percentual: 15 %%\n");
    }else if(salario > 400 && salario <=800){
        novoSalario = salario + (salario * (12.0/100.0));
        aumento = salario * (12.0/100.0);
        printf("Novo salario: %.2f\n",novoSalario);
        printf("Reajuste ganho: %.2f\n", aumento);
        printf("Em percentual: 12 %%\n");
       
    }else if(salario > 800 && salario <= 1200){
        novoSalario = salario + (salario * (10.0/100.0));
        aumento = salario * (10.0/100.0);
        printf("Novo salario: %.2f\n",novoSalario);
        printf("Reajuste ganho: %.2f\n", aumento);
        printf("Em percentual: 10 %%\n");
        
    }else if(salario > 1200 && salario <= 2000){
        novoSalario = salario + (salario * (7.0/100.0));
        aumento = salario * (7.0/100.0);
        printf("Novo salario: %.2f\n",novoSalario);
        printf("Reajuste ganho: %.2f\n", aumento);
        printf("Em percentual: 7 %%\n");
        
    }else{
        novoSalario = salario + (salario * (4.0/100.0));
        aumento = salario * (4.0/100.0);
        printf("Novo salario: %.2f\n",novoSalario);
        printf("Reajuste ganho: %.2f\n", aumento);
        printf("Em percentual: 4 %%\n");
       
    }


	return 0;
}

File: Desafios em Python/desafio1048.py
salario = float(input())

if salario <= 400:
    novo_salario = salario + (salario * 0.15)
    aumento = 15
    reajuste = salario * 0.15
    print(f"reajuste {reajuste} novo total {novo_salario}")
elif salario > 400 and salario <= 800:
    novo_salario = salario + (salario * 0.12)
    aumento = 12
    reajuste = salario * 0.12
elif salario > 800 and salario <= 1200:
    novo_salario = salario + (salario * 0.1)
    aumento = 10
    reajuste = salario * 0.1
elif salario > 1200 and salario <= 2000:
    novo_salario = salario + (salario * 0.07)
    aumento = 7
    reajuste = salario * 0.07
else:
    novo_salario = salario + (salario * 0.04)
    aumento = 4
    reajuste = salario * 0.04

print(f"Novo salario: {novo_salario}")
print(f"Reajuste ganho: {reajuste}")
print(f"Em percentual: {aumento} %")
